notebooks/279-photo-maker/279-photo-maker.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.            return encoder_outputs[0], list(encoder_outputs.hidden_states)
</pre><p>is there is plans to use model with output hidden states False?</p><p> </p><p>it is better to handle inside model via config valies (e.g. encoder.config.return_dict) or passing in internal forward only. Creating such boolean inputs you creates additional  inputs in model, while model behaviour does not depends from these inputs, it is better to freeze them inside model with specific values for avoiding confusion from users side</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
done